src/intelligence_layer/examples/qa/retriever_based_qa.py

```diff
@@ -22,10 +22,12 @@ class RetrieverBasedQaInput(BaseModel):
         question: The question to be answered based on the documents accessed
             by the retriever.
         language: The desired language of the answer. ISO 619 str with language e.g. en, fr, etc.
+        explainability_enabled: Whether to generate highlights (using the explainability feature) for the answer; default False for performance reasons.
```

Comment:
```suggestion
        explainability_enabled: Whether to generate highlights (using the explainability feature) for the answer. Defaults to `False` for performance reasons.
```
This would be more in line with our syntax. We still have a `contributing.md` on our list of todos that would talk about these things. This would also have to be changed in the other places as well.

Reply:
Fixed here -> https://github.com/Aleph-Alpha/intelligence-layer-sdk/commit/656537651d0f65728582eaafa987881d90a0118a